src/components/Box.tsx

```diff
@@ -0,0 +1,123 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {YogaNode} from 'yoga-layout-prebuilt';
+
+interface BoxProps {
+	readonly width?: number | string;
+	readonly height?: number | string;
+	readonly minWidth?: number;
+	readonly minHeight?: number;
+	readonly paddingTop?: number;
+	readonly paddingBottom?: number;
+	readonly paddingLeft?: number;
+	readonly paddingRight?: number;
+	readonly paddingX?: number;
+	readonly paddingY?: number;
+	readonly padding?: number;
+	readonly marginTop?: number;
+	readonly marginBottom?: number;
+	readonly marginLeft?: number;
+	readonly marginRight?: number;
+	readonly marginX?: number;
+	readonly marginY?: number;
+	readonly margin?: number;
+	readonly flexGrow?: number;
+	readonly flexShrink?: number;
+	readonly flexDirection?: 'row' | 'row-reverse' | 'column' | 'column-reverse';
+	readonly flexBasis?: string | number;
+	readonly alignItems?: 'flex-start' | 'center' | 'flex-end';
+	readonly justifyContent?:
+	| 'flex-start'
+	| 'center'
+	| 'flex-end'
+	| 'space-between'
+	| 'space-around';
+	readonly textWrap?:
+	| 'wrap'
+	| 'truncate'
+	| 'truncate-start'
+	| 'truncate-middle'
+	| 'truncate-end';
+	readonly unstable__transformChildren?: (x: any) => any;
```

Comment:
Don't use `any`. Use `unknown`.

Reply:
I can actually do better because we know that it's a `ReactNode`.